### ISSUE TITLE
Set HTMLDOC = driver.Document Here I would like to know the source for the Document.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prajaktamoghe/selenium-vba#132]
```
I called Korea who do not speak English .

We use the google translator .

Dim WithEvents HTMLDOC As HTMLDocument

To do this, the variable declaration and

Set HTMLDOC = WebBrowser1.Document

Document and thus assigned to HTMLDOC

Private Sub HTMLDOC_onmouseover ()

And events

Private Function HTMLDOC_onclick () As Boolean

Taking advantage of the event.

Another

Bringing Document of x, y coordinates corresponding control soseul .

Set Link = HTMLDOC.elementFromPoint (X, Y)
debug.? Link.innerHTML

Does the implementation of how to use this way to the driver? Trouble ?

Ikseupeulrowo as are
I would like to know how , if a driver

The bottom line is , using the ikseupeulrowo control

Set HTMLDOC = WebBrowser1.Document

How is this in the driver hanya geojiyo

Namely

Set HTMLDOC = driver.Document

In this way .... Here I would like to know the source for the Document.
```

Original issue reported on code.google.com by `heesung2...@gmail.com` on 24 Jan 2015 at 4:23

**Comment by GoogleCodeExporter** [comment]
```
To register an event:
Not implemented

To get the source of an element:
Set ele = wd.findElementById("id")
src = ele.GetAttribute("innerHTML")

To get the source of the page:
src = wd.PageSource

To get a web element at coordinates:
Set ele = wd.executeScript("return document.elementFromPoint(arguments[0], 
arguments[1]);", Array(215, 325))
```

Original comment by `florentbr` on 24 Jan 2015 at 6:51
- Changed state: **Done**
